docs/application-connector/10-02-application-gateway-troubleshooting.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Application Gateway troubleshooting
```

Comment:
As a user, I don't know the architecture of the AC by heart. I don't know and I don't care what part of the AC is responsible for the errors I'm getting. 
Change the description to something that describes the problem that you're trying to solve, for example: 
`Error when calling a registered service`